src/view/com/feeds/CustomFeed.tsx

```diff
@@ -65,7 +65,8 @@ export const CustomFeed = observer(
         testID={`feed-${item.displayName}`}
         accessibilityRole="button"
         style={[styles.container, pal.border, style]}
-        onPress={() => {
+        onPress={e => {
+          e.preventDefault()
```

Comment:
The `preventDefault` is required for any clickable element within a post that is NOT an `<a>` on web. Otherwise the click/press bubbles up and is handled by the root Post's anchor tag.